src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,44 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private K[] keys;
+    private V[] values;
+    private int size;
+
+    public StorageImpl() {
+        keys = (K[]) new Object[10];
+        values = (V[]) new Object[10];
+        size = 0;
+    }
```

Comment:
```suggestion
    public StorageImpl() {
        keys = (K[]) new Object[10];
        values = (V[]) new Object[10];
    }
```

Давай вынесем число 10 в константу